src/Sylius/Bundle/CoreBundle/Resources/config/validation/ShippingMethod.xml

```diff
@@ -19,5 +19,32 @@
                 <option name="groups">sylius</option>
             </constraint>
         </property>
+        <property name="configuration">
+            <constraint name="Sylius\Bundle\CoreBundle\Validator\Constraints\ChannelCodeCollection">
```

Comment:
The same with changes to shipping methods, it would be great to extract these changes from this PR